### PR TITLE
Replace unused 'under consideration' position with 'blocked'.

### PR DESCRIPTION
**Pull request by hober**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ Positions are not final. We are happy to revisit positions as new information be
...
 - `oppose` - WebKit considers this specification to be harmful in its current state.
-- `under consideration` - WebKit's position on this specification is being worked out and is not yet settled.
 - `not considering` - WebKit does not intend to look at this specification in the near future.
 
 Note that positions on this repository do not reflect implementation status. We might like something we do not get around to implement for one reason or another. And we might dislike something we nevertheless have to implement.
 
 #### Other possible issue states
 
-- `invalid` - The issue is not about a specification which would be implemented in a browser engine such as WebKit.
+- `blocked` - Coming to a position is blocked on issues identified with the spec or proposal.
```
Comment (marcoscaceres) [inline README.md:26]: Maybe:

```suggestion
- `blocked` - assigning a position is blocked on issues identified with the spec or proposal.
```

File: README.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ Positions are not final. We are happy to revisit positions as new information be
...
 - `duplicate` - The issue is a duplicate of another issue.
+- `invalid` - The issue is not about a specification which would be implemented in a browser engine such as WebKit.
```
Comment (marcoscaceres) [inline README.md:28]:
```suggestion
- `invalid` - The issue is not about a specification that would be implemented in a browser engine such as WebKit.
```

**Review by marcoscaceres — APPROVED** [review]
Nits, but otherwise lgtm

File: README.md (hunk 1 above)
Comment (annevk) [inline README.md:26]:
```suggestion
- `blocked` - Coming to a position is blocked on issues identified with the specification or proposal.
```
(In a prior PR I removed this abbreviation throughout.)

**Review by annevk — APPROVED** [review]
Nice!

**Pull request merged**